zaza/controller.py

```diff
@@ -39,7 +40,12 @@ async def async_add_model(
     await controller.connect()
     logging.debug("Adding model {}".format(model_name))
     model = await controller.add_model(
-        model_name, config=config, cloud_name=cloud_name, region=region)
+        model_name,
+        config=config,
+        cloud_name=cloud_name,
+        credential_name=credential_name,
```

Comment:
Normally, I'd suggest that this was:

```python
credential_name=credential_name or None,
```

to force it to None if it is "not-truthy" to be really explicit about what is passed, but I've checked the python-libjuju and it only does a truthy check on credential_name anyway.  Oh well.

Reply:
That's a good suggestion! I was mostly following the layout of the other parameters :smile:

Reply:
> That's a good suggestion! I was mostly following the layout of the other parameters 😄

And I think that's really reasonable as well! :-)